eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/RatifySpec.hs

```diff
@@ -35,6 +36,140 @@ spec =
+            modifyPParams $
+              ppDRepVotingThresholdsL
+                .~ (drepVotingThresholds & dvtCommitteeNormalL .~ threshold)
+          enactThreshold ::
+            UnitInterval ->
+            Credential 'DRepRole (EraCrypto era) ->
+            Credential 'HotCommitteeRole (EraCrypto era) ->
+            ImpTestM era ()
+          enactThreshold threshold drepC hotCommitteeC = do
+            drepVotingThresholds <- getsPParams ppDRepVotingThresholdsL
+            let paramChange =
+                  ParameterChange
+                    SNothing
+                    ( emptyPParamsUpdate
+                        & ppuDRepVotingThresholdsL
+                          .~ SJust (drepVotingThresholds & dvtCommitteeNormalL .~ threshold)
+                    )
+                    SNothing
+            pcGai <- submitGovAction paramChange
+            submitYesVote_ (DRepVoter drepC) pcGai
+            submitYesVote_ (CommitteeVoter hotCommitteeC) pcGai
+            passNEpochs 2
+          submitTwoExampleProposalsAndVoteOnTheChild ::
+            GovPurposeId 'CommitteePurpose era ->
+            KeyHash 'DRepRole (EraCrypto era) ->
+            ImpTestM era (GovActionId (EraCrypto era))
+          submitTwoExampleProposalsAndVoteOnTheChild gpiCC drepKH = do
+            committeeC <- KeyHashObj <$> freshKeyHash
+            let updateCC parent = UpdateCommittee parent mempty (Map.singleton committeeC $ EpochNo 5) $ 1 %! 2
+            gaiParent <- submitGovAction $ updateCC $ SJust gpiCC
+            -- We submit a descendent proposal so that even though it is sufficiently
+            -- voted on, it cannot be ratified before the ParameterChange proposal
+            -- is enacted.
+            gaiChild <- submitGovAction $ updateCC $ SJust $ GovPurposeId gaiParent
+            submitYesVote_ (DRepVoter $ KeyHashObj drepKH) gaiChild
+            passEpoch -- Make the votes count
+            pure gaiChild
+      it "Increasing the threshold prevents a hitherto-ratifiable proposal from being ratified" $ do
+        (drepC, hotCommitteeC, gpiCC) <- electBasicCommittee
+        setThreshold smallerThreshold
+        (drepKH, _stakingKH, _paymentKP) <- setupSingleDRep 1_000_000
+        gai <- submitTwoExampleProposalsAndVoteOnTheChild gpiCC drepKH
+        isDRepAccepted gai `shouldReturn` True
+        enactThreshold largerThreshold drepC hotCommitteeC
+        isDRepAccepted gai `shouldReturn` False
+      it "Decreasing the threshold ratifies a hitherto-unratifiable proposal" $ do
+        (drepC, hotCommitteeC, gpiCC) <- electBasicCommittee
+        setThreshold largerThreshold
+        (drepKH, _stakingKH, _paymentKP) <- setupSingleDRep 1_000_000
+        gai <- submitTwoExampleProposalsAndVoteOnTheChild gpiCC drepKH
+        isDRepAccepted gai `shouldReturn` False
+        enactThreshold smallerThreshold drepC hotCommitteeC
+        isDRepAccepted gai `shouldReturn` True
```

Comment:
I guess it would require votes from CC members and SPOs,so maybe it would be an overkill for the test, but it might be worth while. I'll leave the decision up to you

Reply:
It was easier than I thought :+1: I have made both the tests that decrease the threshold to enact the proposals :raised_hands: :smiling_imp: